### ISSUE TITLE
why not using softmax loss function

### ISSUE THREAD

**Issue by auroua** [openai/improved-gan#14]
The supervisied loss function used for mnist feature matching is  
[loss_lab = -T.mean(l_lab) + T.mean(z_exp_lab)](https://github.com/openai/improved-gan/blob/master/mnist_svhn_cifar10/train_mnist_feature_matching.py#L69)
Why not use softmax loss function?

**Comment by zhangqianhui** [comment]
may be the Numerical stability problems
